api/src/middlewares/services/UserService/UserService.ts

```diff
@@ -46,12 +46,12 @@ export class UserService {
     return await this.queueService.sendToQueue("NOTIFY_PROCESS", { answers, metadata });
   }
 
-  async sendEmailToUser(data: { answers: AnswersHashMap; metadata: { reference: string; payment?: PayMetadata; type: FormType } }) {
+  async sendEmailToUser(data: { answers: AnswersHashMap; metadata: { reference: string; payment?: PayMetadata; type: FormType; postal?: boolean } }) {
     const { answers, metadata } = data;
     const { reference, type } = data.metadata;
 
-    const personalisation = PersonalisationBuilder.userConfirmation(answers, metadata);
-    const templateName = getUserTemplate(answers.country as string);
+    const templateName = getUserTemplate(answers.country as string, metadata.postal);
+    const personalisation = PersonalisationBuilder[templateName](answers, metadata);
```

Comment:
```suggestion
    const templateName = getUserTemplate(answers.country as string, metadata.postal);
    const personalisationBuilderForTemplate = PersonalisationBuilder[templateName]
    const personalisation = personalisationBuilderForTemplate(answers, metadata);
```

declare intermediate variable for readability